Xamarin.Forms.Platform.UAP/BoxViewRenderer.cs

```diff
@@ -48,10 +49,22 @@ protected override AutomationPeer OnCreateAutomationPeer()
 			return new FrameworkElementAutomationPeer(Control);
 		}
 
+		protected override void UpdateBackgroundColor()
+		{
+			//background color change must be handled separately
+			//because the background would protrude through the border if the corners are rounded
+			//as the background would be applied to the renderer's FrameworkElement
+			Color backgroundColor = Element.BackgroundColor;
+			if (Control != null)
+			{
+				Control.Background = backgroundColor.IsDefault ? null : backgroundColor.ToBrush();
+			}
```

Comment:
Here you can use early return pattern, to validate if control is null..
eg:

```C#
			if (Control != null) return;
			
			//background color change must be handled separately
			//because the background would protrude through the border if the corners are rounded
			//as the background would be applied to the renderer's FrameworkElement
			Control.Background = backgroundColor.IsDefault ? null : Element.BackgroundColor.ToBrush();
```

Reply:
I honestly just copied how the FrameRenderer was handling the background because I ran into the issue of the background color extending beyond the border.  Mainly a style preference, but I'm ok with changing that. 

In your example it looks like the Element.BackgroundColor property would be called twice, you want to avoid that since bindable properties are essentially list a lookup

Reply:
Thats true! you have a good point about bindable properties!